### DISCUSSION TITLE
Filtering with decimal type

### DISCUSSION THREAD

**Discussion by sh-sharifjonov** [alirezanet/Gridify#166] [Q&A] [answered]
I have a column with the decimal type. When applying filtering, conversion to this type generates an error if the value is floating point. For example: the numbers 1, 2... are converted well, but if it is 2.00 or 3.00 an error is generated.
```
        AddMap("TotalPrice", o => o.Price, p => decimal.Parse(p, CultureInfo.InvariantCulture));
        AddMap("Commission", o => o.CommissionValue);
```
If you use a mapper and add a converter, then everything will work well. But if you do not add a converter, a conversion error is displayed. I don't want to add a converter and convert the value every time. Is there any solution here? Is it possible to disable automatic conversion so that LINQ itself converts the value?

**Comment by alirezanet** [comment, marked as answer]
Hi @ShuhratSharifjonovEskhata,
the alternative solution would be modifying the operating system decimal configuration, in most cases for example separator is different in OS level depending on your region.

![image](https://github.com/alirezanet/Gridify/assets/7004080/25927294-df4f-4329-9fcd-d7c57791f673)
